python/api/service/serializers.py

```diff
@@ -52,6 +52,21 @@ def create(self, validated_data):
             raise AdcmEx('SERVICE_CONFLICT') from None
 
 
+class ServiceUISerializer(ServiceSerializer):
+    action = CommonAPIURL(read_only=True, view_name='object-action')
+    name = serializers.CharField(read_only=True)
+    version = serializers.SerializerMethodField()
+    concerns = ConcernItemUISerializer(many=True, read_only=True)
+    locked = serializers.BooleanField(read_only=True)
+    status = serializers.SerializerMethodField()
+
+    def get_version(self, obj):
+        return obj.prototype.version
+
+    def get_status(self, obj):
+        return status_api.get_service_status(obj)
```

Comment:
Explicit import is better `from cm.status_api import get_service_status`

Reply:
fixed